_layouts/default.html

```diff
@@ -61,9 +61,6 @@
         <paper-item label="Developer API (javadoc)" icon="open-in-browser">
           <a href="api/latest/" target="_blank"></a>
         </paper-item>
-        <paper-item label="Releases &amp; Versioning">
```

Comment:
Can you just run the at-head moeified tar build target, untar it, and see if it's missing there?  I am curious if it's just an artifact of syncing, as you say, or if there's something going wrong and it's being blown up otherwise.